vue-app/src/views/AboutHowItWorks.vue

```diff
@@ -53,8 +53,9 @@
     <ul>
       <!-- TODO pull in variable here -->
       <li>This phase will last xxx days.</li>
```

Comment:
@samajammin trying to figure out what to do here. Dont have a good way of figuring out how long the join phase will be, just how long would be left based on the start time. Any suggestions here?

Reply:
Good point, we should just remove it. We won't know how long the join phase is.

Reply:
done!